assignment1.Rmd

```diff
@@ -16,18 +16,31 @@ knitr::opts_chunk$set(echo = FALSE) # actually set the global chunk options.
 
 ### Exercise 1
 
-Here, I will present the results of writing a for loop to print out the numbers from 1 to 10.
+Here, I will present the results of writing a for loop to print out the numbers from 1 to 20.
```

Comment:
Dan, what are you doing here? You've doubled our workload!

Reply:
No that's all good, best to be thorough!